velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -119,7 +119,28 @@ TEST_F(StringAggregatesTest, nonFixedWidthAggregate) {
   })};
 
   testWindowFunction(input, "min(c2)", kOverClauses);
-  testWindowFunction(input, "max(c2)", kOverClauses);
+  testWindowFunction(input, "max(c2)", kOverClauses, {""}, false);
+}
+
+class DecimalAggregatesTest : public WindowTestBase {};
+
+TEST_F(DecimalAggregatesTest, decimalWindowAggregate) {
+  auto testAggregate = [&](const TypePtr& type, vector_size_t size) {
+    auto input = {makeRowVector({
+        makeRandomInputVector(BIGINT(), size, 0.2),
+        makeRandomInputVector(type, size, 0.2),
+        makeRandomInputVector(BIGINT(), size, 0.0),
+        makeRandomInputVector(BIGINT(), size, 0.0),
+    })};
+
+    testWindowFunction({input}, "min(c1)", kOverClauses);
+    testWindowFunction({input}, "max(c1)", kOverClauses, {""}, false);
```

Comment:
Realized later that instead of {""}, we should use kFrameClauses for the invocations here.

Reply:
Thanks for the suggestion, updated. Only the default frame clause was used earlier to reduce the test time. c2 and c3 are also updated to contain positive integers.